tests/test_IOB.py

```diff
@@ -0,0 +1,127 @@
+import unittest
+import insulin_on_board
+
+class IOB_Tests(unittest.TestCase):
+    
+    def test_format_normal(self):
+        self.input = [ {"deviceId": "DemoData-123456789",
```

Comment:
Since your fixtures (expected inputs and outputs) are different across different tests, you shouldn't be attaching them to the instance with `self.`